app/src/main/java/com/gh4a/BaseActivity.java

```diff
@@ -96,6 +100,10 @@
     private View mLeftDrawerHeader;
     private View mRightDrawerHeader;
     private SupportMenuInflater mMenuInflater;
+    private ImageView mNotificationsIndicator;
+
+    private Drawable mTintedCircleIcon;
+    private Drawable mCircleIcon;
```

Comment:
My gut feeling is that all of the changes in this file should be in HomeActivity. None of this is relevant to any other activity.